src/i18n/en.json

```diff
@@ -19,12 +19,11 @@
   "No data to display...": "No data to display...",
   "No emoji found": "No emoji found",
   "Objects": "Objects",
-  "People & Body": "People & Body",
   "Pick your emoji": "Pick your emoji",
   "Post": "Post",
   "Search": "Search",
   "Search Results": "Search Results",
-  "Smileys & Emotion": "Smileys & Emotion",
+  "Smileys & People": "Smileys & People",
```

Comment:
is this change necessary to adapt to emoji-mart default set?

Reply:
@mahboubii the intention was not only to align it with the npm package, but also with the key under which it is stored in the I18n object (`categories.people`). Do you prefer me to revert this change?

Reply:
reverting back to Smileys & Emotion